### PR TITLE
fix(ModelHelpers): add overload for converting model meshes using bui…

### PR DESCRIPTION
**Pull request by BotellaA**
…lders

### CODE REVIEW

File: include/geode/model/helpers/convert_model_meshes.h [hunk 1]
```diff
@@ -28,21 +28,33 @@
...
+    class BRepBuilder;
     class Section;
+    class SectionBuilder;
 } // namespace geode
 
 namespace geode
 {
     void opengeode_model_api convert_surface_meshes_into_triangulated_surfaces(
         BRep& brep );
+    void opengeode_model_api convert_surface_meshes_into_triangulated_surfaces(
```
Comment (BotellaA) [inline include/geode/model/helpers/convert_model_meshes.h:40]: Should we deprecate the other one? 
I think both should be kept.

File: include/geode/model/representation/core/brep.h [hunk 2]
```diff
@@ -67,6 +67,8 @@ namespace geode
                                      public Identifier
     {
     public:
+        static constexpr index_t dimension{ 3 };
```
Comment (BotellaA) [inline include/geode/model/representation/core/brep.h:70]: This is very handy for template progamming

File: include/geode/model/helpers/convert_model_meshes.h (hunk 1 above)
Comment (panquez) [reply include/geode/model/helpers/convert_model_meshes.h:40]: I agree

**Review by panquez — APPROVED** [review]

**Pull request merged**